ui/packages/ui/src/app/pages/createConnector/connectorSteps/SelectConnectorTypeComponent.tsx

```diff
@@ -46,6 +67,17 @@ export const SelectConnectorComponent: React.FunctionComponent<ISelectConnectorC
       });
   }, [setConnectorTypes]);
 
+  const onClick = (event: { currentTarget: { id: string; }; }) => {
+    // The id is the connector className
+    const newId = event.currentTarget.id;
+    const selectedConn = connectorTypes.find(cType => cType.className === newId);
+
+    // Set selection undefined if connector is not enabled or no change in type (deselection)
```

Comment:
I think we should not perform any action for disabled connector type. (e.g disable click for those cards)